src/components/VueBootstrapTypeahead.vue

```diff
@@ -16,7 +16,7 @@
         @focus="isFocused = true"
         @blur="handleBlur"
         @input="handleInput($event.target.value)"
-        @keyup.esc="handleEsc($event.target.value)"
+        @keydown.esc="handleEsc($event.target.value)"
```

Comment:
The bug here is that there are 2 handlers being called when the dropdown is open and ESC is pressed:

- one to close the dropdown (via `listItemBlur` event)
- one to clear the input value (via native input blur event). 

With this change `@keydown` here is only called when pressing ESC while the dropdown is closed resulting in the behaviour we are after. This works because each list item component has the following handler within it:  `@keydown.esc.stop.prevent="$emit('listItemBlur')"` Having the above as `@keyup` would not stop it from being called as originally intended.